test/testcafe/framework/page-objects/FactorEnrollPasswordPageObjectV3.js

```diff
@@ -0,0 +1,24 @@
+import { Selector } from 'testcafe';
+
+import EnrollPasswordPageObject from './FactorEnrollPasswordPageObject';
+
+const SWITCH_AUTHENTICATOR_LINK = '[data-se="switchAuthenticator"]';
+
+/**
+ * This page object will be used by 
+ * password enrollment
+ * password expiry
+ * password will expire soon 
+ * admin initiated password reset scenarios.
+ * 
+ * TODO: Rename this to have AuthenticatorPasswordPageObject when Factor is cleaned up.
```

Comment:
can you expand this comment further so we know what "Factor is cleaned up" means? Or file a JIRA in tech debt and link it to this

Reply:
Actually this was copied from Lester's PR 😅  I will remove this.